test/controllers/users_controller_test.rb

```diff
@@ -10,7 +10,7 @@ class UsersControllerTest < ActionDispatch::IntegrationTest
 
   describe 'create method' do
     it 'should raise if user is not admin' do
-      User::ROLE.reject { |role| role == :admin }.each do |role|
+      User.role.values.reject { |role| role == :admin }.each do |role|
```

Comment:
Enumeration is basically the way of doing a few things. It can bound a set of values, for one. For example, the data type `boolean` is an enum, it has two constant values, `true` and `false`. For most programming you'll ever see, enums will be bit strings, basically like:
```
0=THIS
1=THAT
2=THAR
```

These become useful if you want to deliberately limit the input to a system via code. It also becomes helpful because rather than save something like "1" to a DB via code, you can go "THAT" so it reads cleaner. 

Here's a fun read through on that: https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html

Much easier to switch on MONDAY, TUESDAY, WEDNESDAY, when you could also just do 1,2,3. Reads cleaner for all parties involved.